ui/dashboards/src/components/Panel/PanelActions.tsx

```diff
@@ -236,7 +253,9 @@ export const PanelActions: React.FC<PanelActionsProps> = ({
         </OnHover>
         {divider} {queryStateIndicator}
         <OnHover>
-          {extraActions} {readActions}
+          {extraActions}
+          {viewQueryAction}
```

Comment:
This probably should be inside the overflow menu, we have a lot of actions and they are included in a menu for smaller screens